app/src/main/java/org/mozilla/focus/screenshot/ScreenshotCaptureTask.java

```diff
@@ -0,0 +1,74 @@
+package org.mozilla.focus.screenshot;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.os.AsyncTask;
+import android.text.TextUtils;
+
+import org.mozilla.focus.screenshot.model.Screenshot;
+import org.mozilla.focus.utils.FileUtils;
+import org.mozilla.focus.utils.StorageUtils;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class ScreenshotCaptureTask extends AsyncTask<Object, Void, String> {
+
+    private final Context context;
+
+    public ScreenshotCaptureTask(Context context) {
```

Comment:
I guess Context.getApplicationContext() would make things easier if you want the ApplicationContext

Reply:
This is invalid, sorry!